lib/active_model/serializer.rb

```diff
@@ -2,10 +2,12 @@
 
 module ActiveModel
   class Serializer
+    Error = Class.new(StandardError)
     extend ActiveSupport::Autoload
     autoload :Configuration
     autoload :ArraySerializer
     autoload :Adapter
+    autoload :Builder
```

Comment:
Now that I look at it, shouldn't we be moving away from `autoload` to a thread safe world?